packages/flutter_test/lib/src/finders.dart

```diff
@@ -23,18 +23,47 @@ const CommonFinders find = CommonFinders._();
 class CommonFinders {
   const CommonFinders._();
 
-  /// Finds [Text] and [EditableText] widgets containing string equal to the
-  /// `text` argument.
+  /// Finds [Text], [EditableText], and optionally [RichText] widgets
+  /// containing string equal to the `text` argument.
+  ///
+  /// If `skipRichText` is disabled, [RichText] widgets (and therefore also
+  /// [Text] and [Text.rich] widgets) are matched by comparing the
+  /// [InlineSpan.toPlainText] with the given `text`.
+  /// If `skipRichText` is enabled, all standalone [RichText] widgets are
+  /// ignored and [Text] matching shortcuts to [Text.data].
```

Comment:
This sounds confusing, maybe better:

```suggestion
  /// ignored and `text` is matched with [Text.data].
```